tuna/find_db.py

```diff
@@ -138,6 +141,8 @@ class ConvolutionFindDB(BASE, FindDBMixin):  #pylint: disable=too-many-instance-
   @orm.reconstructor
   def __init__(self, **kwargs):
     self.logger = kwargs['logger'] if 'logger' in kwargs else None  #pylint: disable=multiple-statements
```

Comment:
`self.logger = kwargs["logger"] if "logger" in kwargs else setup_logger("find_db")`